### PR TITLE
[Feat] Add Support Color Variant in Style

### PR DESCRIPTION
**Pull request by anggerpradana**
support add text/bg color with color variant (eg; blue_500, blue_100) using magic call.
```
style('text')->text_blue_300()->bg_green_100();
```

using prefix `text_` or `bg_` follow by supported color variant.

see [ColorVariant.php](https://github.com/SonyPradana/php-library/blob/master/src/System/Console/Style/ColorVariant.php)

### CODE REVIEW

File: src/System/Console/Style/Style.php [hunk 1]
```diff
@@ -9,6 +9,7 @@
 use System\Console\Style\Color\ForegroundColor;
 use System\Console\Traits\CommandTrait;
 use System\Text\Str;
+use System\Text\Text;
```
Comment (SonyPradana) [inline src/System/Console/Style/Style.php:12]: remove un use import

**Review by SonyPradana — CHANGES_REQUESTED** [review]
please fix coding style

File: src/System/Console/Style/Style.php (hunk 1 above)
Comment (anggerpradana) [reply src/System/Console/Style/Style.php]: done

**Review by SonyPradana — APPROVED** [review]
thank you

**Pull request merged**